src/colejorz/pilothouse.py

```diff
@@ -130,10 +133,10 @@ def adjust_speed(self, level, timed=0):
                     # new instruction - stop doing this one, and exit
                     return True
                 sleep(0.2)
-            return self.change_speed(0)
+            return self.change_speed(0)  # type: ignore
```

Comment:
`change_speed()` always  returns `None` - should it be casted to `bool` here or just invoke `change_speed()` and return False?

Reply:
Either
```
self.change_speed(0)
return False
```
or
```
self.change_speed(0)
return None
```

But why would we want to distinguish these False and None? It would need some docstring changes. As currently documented, it shouldn't be `Optional[bool]`.

This adds a place for a comment if it's not clear if we haven't finished the speed adjustment while stopping.